### PR TITLE
feat: introduce a generic way to configure links

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Instead of spreading different options to configure link title/href, I introduced a generic way in which Che/CRW is able to fully to override links since they have a little in common.

During testing just run and make sure everything works as previously.

CRW is adapted with https://github.com/redhat-developer/codeready-workspaces-images/pull/53

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-1879

### CODE REVIEW

**Review by JPinkney — APPROVED** [review]

File: packages/dashboard-frontend/src/Layout/Header/Tools/__tests__/index.spec.tsx [hunk 1]
```diff
@@ -151,7 +150,7 @@ describe('Page header tools', () => {
     const infoButton = screen.getByRole('button', { name: 'info button' });
     fireEvent.click(infoButton);
 
-    const helpItem = screen.getByRole('menuitem', { name: new RegExp(helpTitle, 'i') });
+    const helpItem = screen.getByRole('menuitem', { name: new RegExp("Community", 'i') });
```
Comment (olexii4) [inline packages/dashboard-frontend/src/Layout/Header/Tools/__tests__/index.spec.tsx:153]: A minor proposal
"Community" -> 'Community'

File: packages/dashboard-frontend/src/Layout/Header/Tools/index.tsx [hunk 2]
```diff
@@ -204,7 +204,7 @@ export class HeaderTools extends React.PureComponent<Props, State> {
                 Click here
               </Button>
               <span> to see the login command and copy it manually.</span>
-              <pre className="refresh-token-area">{loginCommand}</pre>
+              <pre className='refresh-token-area'>{loginCommand}</pre>
```
Comment (olexii4) [inline packages/dashboard-frontend/src/Layout/Header/Tools/index.tsx:207]: As for me, I used " instead of ' in HTML. 
@akurinnoy WDYT?

**Review by olexii4 — APPROVED** [review]
LGTM

**Comment by dmytro-ndp** [comment]
[crw-ci-test]

File: packages/dashboard-frontend/src/Layout/Header/Tools/index.tsx (hunk 2 above)
Comment (akurinnoy) [reply packages/dashboard-frontend/src/Layout/Header/Tools/index.tsx:207]: yes, we use double quotes in HTML and single quotes in JS/TS.

**Review by akurinnoy — APPROVED** [review]

File: packages/dashboard-frontend/src/Layout/Header/Tools/__tests__/index.spec.tsx (hunk 1 above)
Comment (sleshchenko) [reply packages/dashboard-frontend/src/Layout/Header/Tools/__tests__/index.spec.tsx]: fixed

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**